osu.Game/Overlays/Changelog/ChangelogHeader.cs

```diff
@@ -39,6 +39,8 @@ public ChangelogHeader()
 
             Build.ValueChanged += showBuild;
 
+            // ReSharper disable once PossibleNullReferenceException
+            // see https://youtrack.jetbrains.com/issue/RSRP-486768
```

Comment:
Thoughts on these? Still unsure if this is to be considered a r# bug or "out of scope" of its heuristic detection.

Reply:
Why disable it and not put in `CreateContent()`, where it's actually constructed?

Reply:
There are multiple ways around it, for sure. I'm not happy with this PR in its existing state.

More curious about whether that jetbrains report should even exist.

Reply:
Only reported it for the sole reason of suddenly showing up as if it's a regression on their side. Will close that report as it's a bad pattern anyways.